package/MDAnalysis/topology/TPRParser.py

```diff
@@ -157,7 +157,11 @@
 __author__ = "Zhuyi Xue"
 __copyright__ = "GNU Public Licence, v2"
 
-import xdrlib
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import xdrlib
```

Comment:
Please keep this in, the xdrlib deprecation is not an internal warning and shouldn't be filtered out (it's something we'll fix soon on our own).